sound/soc/intel/boards/Kconfig

```diff
@@ -498,4 +498,22 @@ config SND_SOC_INTEL_TGL_RT1308_MACH
 
 endif ## SND_SOC_SOF_TIGERLAKE
 
+if (SND_SOC_SOF_JASPERLAKE || SND_SOC_SOF_ICELAKE) && SND_SOC_SOF_HDA_LINK
+
+config SND_SOC_INTEL_JSL_DA7219_MAX98373_MACH
+	tristate "JSL with DA7219 and MAX98373 in I2S Mode"
```

Comment:
i'd like to have two Kconfigs that use the same common parts, see e.g. how it's done for bxt_da7219_mx98753a

Reply:
Ack